pkgs/build-support/build-sandbox/default.nix

```diff
@@ -52,25 +58,39 @@ in stdenv.mkDerivation ({
     echo '#include "setup.h"' > params.c
     echo 'bool setup_app_paths(void) {' >> params.c
 
-    for dep in $runtimeDeps; do
-      echo 'if (!bind_mount("'"$dep"'", true, true, true)) return false;' \
+    ${if fullNixStore then /* TODO: last two bool arguments correct? */''
```

Comment:
> last two bool arguments correct?

Yes, essentially we **do** want `NOSUID`/`NODEV` on these directories and canonicalization doesn't hurt either.